### PR TITLE
refactor(vehicle_cmd_gate)!: delete rate limit skipping function for vehicle departure

### PR DESCRIPTION
**Pull request by yuki-takagi-66**
## Description
This PR delete rate limit skipping function which is activated when the vehicle departure.
More appropriate feature, including stopping, can be achieved by the parameter settings.

## Related links
https://github.com/autowarefoundation/autoware_launch/pull/1077

## How was this PR tested?
I have tested this change by two type of real vehicles.

## Notes for reviewers

None.

## Effects on system behavior

None.

### CODE REVIEW

File: control/autoware_vehicle_cmd_gate/config/vehicle_cmd_gate.param.yaml [hunk 1]
```diff
@@ -16,14 +16,14 @@
...
-      lon_acc_lim: [5.0, 4.0]
-      lon_jerk_lim: [5.0, 4.0]
-      lat_acc_lim: [5.0, 4.0]
-      lat_jerk_lim: [7.0, 6.0]
-      actual_steer_diff_lim: [1.0, 0.8]
+      reference_speed_points: [0.1, 0.3, 20.0, 30.0]
+      steer_lim: [1.0, 1.0, 1.0, 0.8]
+      steer_rate_lim: [1.0, 1.0, 1.0, 0.8]
+      lon_acc_lim: [5.0, 5.0, 5.0, 4.0]
+      lon_jerk_lim: [80.0, 5.0, 5.0, 4.0]
```
Comment (TakaHoribe) [inline control/autoware_vehicle_cmd_gate/config/vehicle_cmd_gate.param.yaml:23]: Please add some comments here to describe the reasons/backgrounds of this parameter.

Comment (yuki-takagi-66) [reply control/autoware_vehicle_cmd_gate/config/vehicle_cmd_gate.param.yaml]: Thank you. I added a brief comments

Comment (yuki-takagi-66) [reply control/autoware_vehicle_cmd_gate/config/vehicle_cmd_gate.param.yaml]: @TakaHoribe 
Thank you. I added a brief comments

Comment (yuki-takagi-66) [reply control/autoware_vehicle_cmd_gate/config/vehicle_cmd_gate.param.yaml]: @TakaHoribe 
I also add some comments to the readme document

File: control/autoware_vehicle_cmd_gate/config/vehicle_cmd_gate.param.yaml [hunk 2]
```diff
@@ -16,14 +16,14 @@
...
-      lon_acc_lim: [5.0, 4.0]
-      lon_jerk_lim: [5.0, 4.0]
-      lat_acc_lim: [5.0, 4.0]
-      lat_jerk_lim: [7.0, 6.0]
-      actual_steer_diff_lim: [1.0, 0.8]
+      reference_speed_points: [0.1, 0.3, 20.0, 30.0]
+      steer_lim: [1.0, 1.0, 1.0, 0.8]
+      steer_rate_lim: [1.0, 1.0, 1.0, 0.8]
+      lon_acc_lim: [5.0, 5.0, 5.0, 4.0]
+      lon_jerk_lim: [80.0, 5.0, 5.0, 4.0] # The first element is required for quick pedal changes when stopping and starting on a slope.
```
Comment (TakaHoribe) [inline control/autoware_vehicle_cmd_gate/config/vehicle_cmd_gate.param.yaml:23]: This setting is not unique for a slope.
```suggestion
      lon_jerk_lim: [80.0, 5.0, 5.0, 4.0] # The first element is required for quick pedal changes when stopping and starting.
```

**Review by TakaHoribe — APPROVED** [review]
LGTM, but please check my suggestion before merge.

**Pull request merged**